public/app/features/panel/panellinks/linkSuppliers.ts

```diff
@@ -1,8 +1,31 @@
 import { PanelModel } from 'app/features/dashboard/state/PanelModel';
-import { FieldDisplay, ScopedVars, DataLinkBuiltInVars } from '@grafana/ui';
-import { LinkModelSupplier, getTimeField } from '@grafana/data';
+import { FieldDisplay, ScopedVars, ScopedVar } from '@grafana/ui';
+import { LinkModelSupplier, getTimeField, Labels } from '@grafana/data';
 import { getLinkSrv } from './link_srv';
 
+interface SeriesVars {
+  name: string;
+  labels: Labels;
+  refId?: string;
+}
+
+interface FieldVars {
+  name: string;
+}
+
+interface ValueVars {
+  raw: number;
```

Comment:
@ryantxu do you think we need this, or `numeric` is just fine?

Reply:
raw is any